test/Microsoft.ML.TestFrameworkCommon/CrashTestHostProcessorHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
```

Comment:
I know I had asked you to add this, but on second thoughts, it doesn't seem to add much. It is simply a rename over an existing function. Maybe we can delete this and just use Environment.FailFast directly wherever we would have called CrashTestHostProcessHelper.CrashTestHostProcess()
Maybe we should delete this.

Reply:
Yes, it is just one line function I'm okay with delete this.

---
In reply to: [373198033](https://github.com/dotnet/machinelearning/pull/4744#discussion_r373198033) [](ancestors = 373198033)